projects/ad9172/src/main.c

```diff
@@ -192,6 +206,15 @@ int main(void)
 		goto error_5;
 	}
 
+#ifdef DAC_DMA_EXAMPLE
+	extern const uint32_t sine_lut_iq[1024];
+	axi_dac_load_custom_data(tx_dac, sine_lut_iq,
+				 ARRAY_SIZE(sine_lut_iq),
+				 DDR_MEM_BASEADDR);
+	axi_dmac_init(&tx_dmac, &tx_dmac_init);
+	axi_dmac_transfer(tx_dmac, DDR_MEM_BASEADDR,
+			  sizeof(sine_lut_iq) * 2);
```

Comment:
I guess that this multiplication by 2 is related to the number of channels. I would use that  information instead since it should be available at this point.

Reply:
Well..."num_channels" in "tx_dac_init" is 4, so an other division will be needed.